src/order/complete_lattice.lean

```diff
@@ -42,6 +42,32 @@ class complete_lattice (α : Type u) extends bounded_lattice α, has_Sup α, has
 (Inf_le : ∀s, ∀a∈s, Inf s ≤ a)
 (le_Inf : ∀s a, (∀b∈s, a ≤ b) → a ≤ Inf s)
 
+/-- Create a `complete_lattice` from a `partial_order` and `Inf` function
+that returns the greatest lower boundary of a set. Usually this constructor provides
```

Comment:
```suggestion
that returns the greatest lower bound of a set. Usually this constructor provides
```

Reply:
I made this typo again :(

Reply:
We all make typos depending on our mother tongue. You have the right to use boundary instead of bound, and to do whatever mess you like with articles.